diffractometer.rst

```diff
@@ -0,0 +1,52 @@
+Diffractometer API
+~~~~~~~~~~~~~~~~~~
+
+**API Functions**
+
+.. code:: python
+
+    # ActuatorData as defined in beamline
+    from beamline import ActuatorData
+
+    # Decision Needed !
+    # We need to discuss wether this method is needed or not
+    # could ovelap with beamline.get_actuators
+    def get_motors() -> Dict[str, ActuatorData]:
+    """
+    :returns: A dictionary with all available motors where the key
+              is the motor name and the value the ActuatorData tuple
+    :rtype dict:
+    """
+        pass
+
+
+    # Decision Needed !
+    def get_motor(name:str) -> ActuatorData:
```

Comment:
I would strongly suggest NOT to have Diffractometer commands that are just aliases of Beamline commands, either for actuators or procedures. If you are getting the kappa motor you are getting the kappa motor; you should not have beamline.get_actuator('kappa') and diffractometer.get_motor('kappa') as synonyms to do the same thing. And I think it is more confusing to divide the actuators up than to keep them together. Actuator names should be globally unique, so we do not need namespacing. Having people remember whether 'frontlight' or 'beamstop' is a beamline or diffractometer motor just adds an extra complication, when you could put all of them in beamline.

Commands for getting or setting a group of motors (e.g. alignment motors) is another matter. That could legitimately belong to  some specific context.

Reply:
Its a file which I started already some weeks ago so its outdated and should not have been in the PR. In general I agree with you to now have these as aliases.